pixel-avatar-hardhat/contracts/PixelAvatars.sol

```diff
@@ -49,6 +50,8 @@ contract PixelAvatars is ERC721Enumerable, ReentrancyGuard, Ownable {
         _;
     }
 
+    uint256 public mintPrice = 0.01 ether;
+
```

Comment:
Move this `mintPrice` property next to `_devDaoContract` property and before `validDevDaoToken` modifier. 

By having properties placed right above everything else would make it easier to find smart contract properties